src/consolidate_feature_data.py

```diff
@@ -166,12 +167,14 @@ def consolidate_features_without_project(repositories: List[dict],
 
     # Iterate over all repositories from config file
     for repository in repositories:
-        repo_name = repository["repoName"]
+        repo = Repository(orgName=repository["orgName"],
```

Comment:
This does not make sense.

Reply:
Comment was solved during big OOP refactoring.